component/telemetry.go

```diff
@@ -26,4 +27,7 @@ type TelemetrySettings struct {
 
 	// TracerProvider that the factory can pass to other instrumented third-party libraries.
 	TracerProvider trace.TracerProvider
+
+	// MeterProvider that the factory can pass to other instrumented third-party libraries.
+	MeterProvider metric.MeterProvider
```

Comment:
Let's not add this yet, will add in the next PR :)

Reply:
Removed 👍